OrbitGl/TracepointTrack.h

```diff
@@ -26,6 +28,7 @@ class TracepointTrack : public EventTrack {
 
  private:
   [[nodiscard]] bool HasTracepoints() const;
+  std::deque<std::pair<float, float>> circle_points;
```

Comment:
std::vector<Vec2>? A vector would do fine here since we don't need to insert or remove elements. Also we have the Vec2 class that can be used for the point coordinates.

Reply:
Thank you @pierricgimmig for pointing this out. I have updated the code.